### PR TITLE
Adding different describes to the test

### PR DESCRIPTION
**Pull request by manuelq12**
Adding nested describes to have a clearer vision of the flow of the test and to run de async test correctly

### CODE REVIEW

File: test/buy-tshirt.spec.ts [hunk 1]
```diff
@@ -3,30 +3,52 @@ import { MenuContentPage, ProductListPage, ProductAddedModalPage,
...
-  const productAddedModalPage: ProductAddedModalPage = new ProductAddedModalPage();
-  const summaryStepPage: SummaryStepPage = new SummaryStepPage();
-  const signInStepPage: SignInStepPage = new SignInStepPage();
-  const addressStepPage: AddressStepPage = new AddressStepPage();
-  const shippingStepPage: ShippingStepPage = new ShippingStepPage();
-  const orderSummaryPage: OrderSummaryPage = new OrderSummaryPage();
-  const bankPaymentPage: BankPaymentPage = new BankPaymentPage();
-  const paymentStepPage: PaymentStepPage = new PaymentStepPage();
-  it('then should be bought a t-shirt', async () => {
+describe('This must open the shopping website in the browser', () => {
```
Comment (yesid-lopez) [inline test/buy-tshirt.spec.ts]: It should follow a Given.. When... Then... structure...Please improve your description

File: test/buy-tshirt.spec.ts [hunk 2]
```diff
@@ -3,30 +3,52 @@ import { MenuContentPage, ProductListPage, ProductAddedModalPage,
...
+      const productAddedModalPage: ProductAddedModalPage = new ProductAddedModalPage();
+      const summaryStepPage: SummaryStepPage = new SummaryStepPage();
+
+      await menuContentPage.goToTShirtMenu();
+      await productListPage.addToCart();
+      await productAddedModalPage.clickModalBtn();
+      await summaryStepPage.clickCheckOut();
+    });
+
+    describe('Then it must login into the website', () => {
```
Comment (yesid-lopez) [inline test/buy-tshirt.spec.ts:23]: It should follow a Given.. When... Then... structure...Please improve your description

File: test/buy-tshirt.spec.ts [hunk 3]
```diff
@@ -3,30 +3,52 @@ import { MenuContentPage, ProductListPage, ProductAddedModalPage,
...
+      beforeAll(async() => {
+        const signInStepPage: SignInStepPage = new SignInStepPage();
+        await signInStepPage.performLogin();
+      });
+
+      describe('After that, it must select the default address', () => {
```
Comment (yesid-lopez) [inline test/buy-tshirt.spec.ts]: It should follow a Given.. When... Then... structure...Please improve your description

File: test/buy-tshirt.spec.ts [hunk 4]
```diff
@@ -3,30 +3,52 @@ import { MenuContentPage, ProductListPage, ProductAddedModalPage,
...
+        beforeAll(async() => {
+          const addressStepPage: AddressStepPage = new AddressStepPage();
+          await addressStepPage.clickCheckOut();
+        });
+
+        describe('Finally, it must pay via bank wire', () => {
```
Comment (yesid-lopez) [inline test/buy-tshirt.spec.ts]: It should follow a Given.. When... Then... structure...Please improve your description

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**